site/content/kapp-controller/_index.html

```diff
@@ -57,106 +73,132 @@ <h2>Basic Usage</h2>
-        {{< highlight yaml "" >}}
-apiVersion: kappctrl.k14s.io/v1alpha1
-kind: App
+simple-app   Reconcile succeeded   9s             10s
+{{< / highlight>}}
+    <p>Discover more in the <a href="https://carvel.dev/kapp-controller/docs/latest/walkthrough/">App walkthrough</a> documentation.</p>    
+    <h3>Package authoring</h3>
+    <p>kapp-controller's Package CR provides a way to share versioned software for Kubernetes 
+      so users can easily discover and consume available software.</p>
+{{< highlight yaml "">}}
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
 metadata:
-  name: simple-app
-  namespace: default
-  creationTimestamp: "2020-12-15T00:51:48Z"
-  generation: 1
-  resourceVersion: "37396"
-  uid: dbbb639d-1f58-4a1e-ade6-39bc22ee6d82
-  finalizers:
-  - finalizers.kapp-ctrl.k14s.io/delete
+  name: simple-app.corp.com.1.0.0
 spec:
-  serviceAccountName: default-ns-sa
-  fetch:
-  - git:
-      url: https://github.com/vmware-tanzu/carvel-simple-app-on-kubernetes
-      ref: origin/develop
-      subPath: config-step-1-minimal
+  refName: simple-app.corp.com
+  version: 1.0.0
+  releaseNotes: |
+    Initial release of the simple app package
+  valuesSchema:
+    openAPIv3:
+      title: simple-app.corp.com values schema
+      examples:
+      - svc_port: 80
+        app_port: 80
+        hello_msg: stranger
+      properties:
+        svc_port:
+          type: integer
+          description: Port number for the service.
+          default: 80
+          examples:
+          - 80
+        app_port:
+          type: integer
+          description: Target port for the application.
+          default: 80
+          examples:
+          - 80
+        hello_msg:
+          type: string
+          description: Name used in hello message from app when app is pinged.
+          default: stranger
+          examples:
+          - stranger
   template:
-  - ytt: {}
-  deploy:
-  - kapp: {}
-status:
-  observedGeneration: 1
-  conditions:
-  - status: "True"
-    type: ReconcileSucceeded
-  friendlyDescription: Reconcile succeeded
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: index.docker.io/k8slt/kctrl-example-pkg@sha256:8ffa7f9352149dba1d539d0006b38eda357917edcdd39b82497a61dab2c27b75
+      template:
+      - ytt:
+          paths:
+          - config/
+      - kbld:
+          paths:
+          - '-'
+          - .imgpkg/images.yml
+      deploy:
+      - kapp: {}
+{{< / highlight>}}
+    <p>Discover more in the <a href="https://carvel.dev/kapp-controller/docs/latest/package-authoring/">package author</a> documentation.</p>   
+    <h3>Package consumption</h3> 
+    <p>kapp-controller's PackageRepository CR adds a collection of Packages to allow software to be easily discovered and 
+      installed.
+    </p>
+{{< highlight yaml "">}}
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageRepository
+metadata:
+  name: simple-package-repository
```

Comment:
we should probably throw in `namespace: something` to indicate that it is added to a namespace